default.py

```diff
@@ -350,6 +357,43 @@ def select_version(self, game_versions):
         else:
             return None
 
+    def has_inputstream_adaptive(self):
+        """Checks if InputStream Adaptive is installed and enabled."""
+        payload = {
```

Comment:
@aqw Agreed, but unfortunately it seems to be the only way to acquire this information.